### ISSUE TITLE
Backup access code during upgrade

### ISSUE THREAD

**Issue by rmackay9** [HefnySco/andruav_android_app#7]
After a new release of the Android app I often find that I need to re-enter my access code.  It would be nice if this could be backed-up and restored so users don't need to re-enter it.

**Comment by HefnySco** [comment]
fixed and published in the coming 5.00.18 version today.

**Comment by rmackay9** [comment]
Yes, just tested and no need to re-enter the access code after the upgrade, thanks!
